saleor/checkout/utils.py

```diff
@@ -445,22 +446,33 @@ def get_prices_of_discounted_specific_product(
     Product must be assigned directly to the discounted category, assigning
     product to child category won't work.
     """
-    line_prices = []
     voucher_info = fetch_voucher_info(voucher)
     discounted_lines: Iterable["CheckoutLineInfo"] = get_discounted_lines(
         lines, voucher_info
     )
-    discounts = discounts or []
+    line_prices = get_base_lines_prices(checkout_info, discounted_lines, discounts)
+
+    return line_prices
+
 
-    for line_info in discounted_lines:
+def get_base_lines_prices(
+    checkout_info: "CheckoutInfo",
+    lines: Iterable["CheckoutLineInfo"],
+    discounts: Optional[Iterable[DiscountInfo]],
+):
+    """Get base total price of provided checkout lines without discount applied."""
```

Comment:
"without voucher discount applied." right? Sale discount is applied.

Reply:
Yes, you are right. Thanks. I will change that.